x-pack/plugins/reporting/server/lib/esqueue/__tests__/worker.js

```diff
@@ -347,12 +347,6 @@ describe('Worker class', function () {
         expect(body._source).to.eql({ excludes: excludedFields });
       });
 
-      it('should search by job type', function () {
-        const { body } = getSearchParams(jobtype);
-        const conditions = get(body, conditionPath);
-        expect(conditions.must).to.eql({ term: { jobtype: jobtype } });
-      });
-
```

Comment:
This test is no longer relevant, because each search should turn up a waiting job for any type